library/src/main/java/com/pokegoapi/util/PokeNames.java

```diff
@@ -17,72 +17,75 @@
 
 import java.io.UnsupportedEncodingException;
 import java.util.Locale;
-import java.util.ResourceBundle;
 import java.util.MissingResourceException;
+import java.util.ResourceBundle;
 
 /**
  * @author Angelo Rüggeberg
+ * @author Nyazuki
```

Comment:
Let's not start doing this because it will end up being a very long list.
The commit history will provide authorship of any edits.